standalone/instructlab-pipelines-with-dsp-on-rhoai-.md

```diff
@@ -0,0 +1,646 @@
+  * Red Hat OpenShift Service Mesh v2 (v3 is not compatible with RHOAI)
+  * Red Hat OpenShift AI
+
+* Teacher and Judge models with a serving endpoint
+    * If already setup you will need the endpoint, api key, and any CA bundles if needed for each model
+    * If setting up your own using these instructions, you will need additional multi-node A100s or L40s for each model
+* SDG taxonomy tree to utilize for Synthetic Data Generation (SDG), see instructions for creating a [taxonomy tree]
+  on how to set up your own taxonomy tree.
+* An OpenShift AI installation, with the Training Operator and KServe components set to `Managed`
+  * A data science project/namespace, in this document this will be referred to as `<data-science-project-name/namespace>`
+* A [StorageClass] that supports dynamic provisioning with [ReadWriteMany] access mode (see step 3 below).
+* An AWS S3 object store. Alternative object storage solutions that are S3-compliant such as Ceph, Nooba and MinIO are also compatible.
+* A locally installed `oc` command line tool to create and manage kubernetes resources.
+* Ilab CLI (or Skopeo/Oras/etc.) for model downloads
+
+[StorageClass]: https://kubernetes.io/docs/concepts/storage/storage-classes/
+[taxonomy tree]: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux_ai/1.2/html/creating_a_custom_llm_using_rhel_ai/customize_taxonomy_tree
+[ReadWriteMany]: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes
+
+## Steps
+
+Before running the training and evaluation steps we must complete the following:
+
+1. [Prepare data and push to object store](#prepare-data-and-push-to-object-store)
+1. [Setting up Judge & Teacher model](#setting-up-judge--teacher-model)
+    * [Deploy a judge model server](#deploy-a-judge-model-server-optional) (Optional)
+    * [Deploy judge model serving details](#deploy-judge-model-serving-details)
+    * [Deploy a teacher model server](#deploy-a-teacher-model-server-optional) (Optional)
+    * [Deploy teacher model serving details](#deploy-teacher-model-serving-details)
+1. [Setup NFS StorageClass](#optional---setup-nfs-storageclass) (Optional)
+1. [Set Up Data Science Pipelines Server and Run InstructLab Pipeline](#set-up-data-science-pipelines-server-and-run-instructLab-pipeline)
+
+### Prepare data and push to object store
+
+Create a tarball with the [granite-7b-starter] model and [taxonomy tree] and push them to your object store.
+
+```bash
+$ mkdir -p s3-data/{model,taxonomy}
+```
+
+Download ilab model repository in s3-data model directory
+```bash
+# You can also use Oras or Skopeo cli tools to download the model
+# If using other tools besides ilab, ensure that filenames are mapped
+# appropriately
+$ ilab model download --repository docker://registry.redhat.io/rhelai1/granite-7b-starter --release 1.2
+$ cp -r <path-to-model-downloaded-dir>/rhelai1/granite-7b-starter/* s3-data/model
+```
+
+Add your taxonomy tree to the `taxonomy` directory
```

Comment:
I think this step is not needed. The pipeline has a parameter with an external git repository containing the taxonomy. @gmfrasca @rimolive can you confirm?